WSL/wsl-config.md

```diff
@@ -249,6 +244,15 @@ These settings are opt-in previews of experimental features that we aim to make
 |:----|:----|:----|:----|
 |`autoMemoryReclaim`| string | disabled | Automatically releases cached memory after detecting idle CPU usage. Set to `gradual` for slow release, and `dropcache` for instant release of cached memory. |
 |`sparseVhd`| bool | false | When set to true, any newly created VHD will be set to sparse automatically. |
+|useWindowsDnsCache**| bool | false | Only applicable when `experimental.dnsTunneling` is set to true. When this option is set to false, DNS requests tunneled from Linux will bypass cached names within Windows to always put the requests on the wire. |
```

Comment:
If we're documenting those as experimental, might be worth adding which version of WSL introduces those settings as well

Reply:
Versioning is done by adding a `*` or `**` and footnoting it below.

Reply:
Sounds like this is resolved.